0-bootstrap/modules/jenkins/scripts/jenkins_gce_startup_script.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+echo "**** Startup Step 1/9: Update apt-get repositories. ****"
+sudo apt-get update
+
+echo "**** Startup Step 2/9: Install Java. ****"
+sudo apt-get install -y default-jdk
+
+echo "**** Startup Step 3/9: Set variables. ****"
+export TERRAFORM_DIR="/opt/terraform/"
+export TERRAFORM_VERSION="0.12.24"
```

Comment:
Shouldn't we be templating this based on
https://github.com/terraform-google-modules/terraform-example-foundation/pull/77#discussion_r453252131

Reply:
Yes! Thank you - I created a template for this now.